internal/proxy/proxy.go

```diff
@@ -259,11 +259,20 @@ func (p *Proxy) https(rw http.ResponseWriter, req *http.Request) {
 func (p *Proxy) connectAndCopy(conn net.Conn, brw *bufio.ReadWriter, req *http.Request, reconnected bool) {
 	logger := req.Context().Value(LogEntryCtx).(*zap.Logger)
 
+	retries := 0
+proxy_dial_retry:
```

Comment:
Go name conventions suggest to use CamelCase for labels too.
https://go.dev/doc/effective_go#mixed-caps

Reply:
changed to ProxyDialRetries